lib/screens/SiteLogsScreen.dart

```diff
@@ -103,8 +103,11 @@ class _SiteLogsScreenState extends State<SiteLogsScreen> {
   loadLogs() async {
     var file = File(widget.site.logFile);
     try {
-      final v = await file.readAsString();
+      String v = await file.readAsString();
```

Comment:
@nbrownus this was one change I was unsure about. I believe I had to convert the variable type from `final` to `String` to allow modifications. Let me know if this looks incorrect to you.